src/app/auth/components/PasswordInput/PasswordInput.tsx

```diff
@@ -48,6 +48,7 @@ const PasswordInput = ({
         min={0}
         required={true}
         autoFocus={autoFocus}
+        autoComplete="new-password"
```

Comment:
avoid assign fixed props to generic components, this could cause wrong behaviours in different parts of the web